core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1018,7 +1019,7 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     val appNameMaxSize = QualOutputWriter.getAppNameSize(Seq(appInfo.get))
     assert(headers.size ==
       QualOutputWriter.getSummaryHeaderStringsAndSizes(appNameMaxSize, 0).keys.size)
-    assert(values.size == headers.size - 1) // unSupportedExpr is empty
+    assert(values.size == headers.size)
```

Comment:
this seems wrong, is unSupportedExpr now not empty?

Reply:
`unSupportedExpr` is still empty, it was originally the last value and as a result would default to empty rather than having an explicit empty value. There a variety of other errors because of some testing infrastructure that doesn't match.